Formula/weather.rb

```diff
@@ -0,0 +1,26 @@
+class Weather < Formula
```

Comment:
name is too generic

Reply:
Should I rename it to something else?

Reply:
Yeah, you should probably talk to upstream about what to call it. Just named "weather," it can't really be included in package managers without that being an unreasonable land grab.

Reply:
Sorry, missed your reply. 

Just to clarify, do you want upstream to suggest a different name for the formula? Or to rename the project?

Reply:
Preferably both :) But only the former is _necessary_